tools/greptimedbcluster/clean-cluster/README.md

```diff
@@ -0,0 +1,69 @@
+# GreptimeDBCluster Cleanup Script
+
+This script is designed to clean up a GreptimeDBCluster and its associated etcd data. It performs the following steps:
+
+1. Deletes a specified GreptimeDBCluster.
+2. Deletes the persistent volume claims (PVCs) associated with that GreptimeDB Datanodes.
+3. Cleans up all data from etcd.
+
+Note: Please proceed with caution when executing this script, as it will permanently delete the GreptimeDBCluster and associated data.
+
+## Prerequisites
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) must be installed.
+- `kubectl` should be configured to access your Kubernetes cluster.
+- Ensure `etcdctl` is installed and can be executed inside the etcd pods.
```

Comment:
Can we use a docker container for etcdctl to skip the installation?

Reply:
Here is `kubectl exec -n "$ETCD_NAMESPACE" "$pod" -- etcdctl xxx` into the etcd pod run etcdctl.
etcdctl is included in etcd pod: https://github.com/etcd-io/etcd/blob/main/Dockerfile#L5
Including bitnami etcd.